docs/api-table.js

```diff
@@ -1999,10 +1999,16 @@ export const iOS = {
     ],
     props: [
       [
-        'icon',
```

Comment:
Has this `icon` prop been deprecated? In that case, maybe it should be informed that it is deprecated and has been replaced by `leadingIcon` and `trailingIcon`. :)

Reply:
Nope it is not deprecated, it is just replaced. No one is using them and if they do they will just get error when compiling.

Reply:
ah alright! 👍